src/es/overlay/energenie-pi-mote.md

```diff
@@ -44,6 +44,6 @@ pin:
 -->
 # Pi-mote
 
-Pi-Mote es una placa transmisora de RF que permite controlar hasta 4 zócalos Energenie de manera independiente utilizando Python.
+Pi-Mote es una placa transSDIra de RF que permite controlar hasta 4 zócalos Energenie de manera independiente utilizando Python.
```

Comment:
Sorry about the previous mess of comments. Here's a place where the find-and-replace changed a word.

Reply:
Oh yeesh, thanks for picking up on this. That'll serve me right for doing case-insentitive search and replace.